### DISCUSSION TITLE
problem allow grant access

### DISCUSSION THREAD

**Discussion by Disqu3-mirOir-qt** [d4rken-org/sdmaid-se#529] [Q&A] [answered]
[Screenshot_20230330-172721.jpg](https://user-images.githubusercontent.com/41585567/228886908-37d9150e-891d-4991-b423-2d0959398dbc.jpg)

**Comment by d4rken** [comment]
What Android version and device is this?

The app SD Maid wants to open HAS to exist, it's a built-in system app.

**Comment by Disqu3-mirOir-qt** [comment]
[Screenshot_20230330-181422.jpg](https://user-images.githubusercontent.com/41585567/228899520-ac0c0440-ad9c-4cfc-afd2-61c56c74895e.jpg)


[Screenshot_20230330-181500.jpg](https://user-images.githubusercontent.com/41585567/228899675-923d4d32-634e-416b-b241-ef0b37057457.jpg)

this file manager used

**Comment by Disqu3-mirOir-qt** [comment]
[Screenshot_20230330-181756.jpg](https://user-images.githubusercontent.com/41585567/228900442-94be512b-9d92-4a44-b448-2b73fd7ab5fa.jpg)

[Screenshot_20230330-181806.jpg](https://user-images.githubusercontent.com/41585567/228900464-0876160c-d5d9-44c9-8d70-dfed710f9aa8.jpg)

i hope all info help you Thanks

**Comment by d4rken** [comment, marked as answer]
The apps `com.android.documentsui` and `com.android.externalstorage` need to exist and be enabled on your device.

They are part of Android since Android 5.0.

**Comment by d4rken** [comment]
I've expanded the setup wiki with details about this:

https://github.com/d4rken-org/sdmaid-se/wiki/Setup#storage-access-framework
